python/test/experimental/distributions/test_multivariate_normal.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+import nnabla as nn
+
+from scipy import stats
+from nnabla.experimental.distributions import MultivariateNormal
+from distribution_test_util import distribution_test_util
+
+
+def test_multivariate_normal():
+    def param_fn(shape):
+        loc = np.random.random(shape)
+        scale = np.random.random(shape) + 1e-5
+        return loc, scale
+
+    def dist_fn(loc, scale):
+        loc = nn.Variable.from_numpy_array(loc)
+        scale = nn.Variable.from_numpy_array(scale)
+        return MultivariateNormal(loc=loc, scale=scale)
+
+    def scipy_fn(loc, scale):
+        return stats.multivariate_normal(np.reshape(loc, (-1,)),
+                                         np.reshape(scale, (-1,)))
```

Comment:
If we create diagonal matrix as covariance such as cov = scale @ scale.T (@ is matrix multiplication), we can delete following 3 functions and make distribution_test_util simpler, can't we?

Reply:
I'm not sure it is true but please try it.